src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -32,33 +32,27 @@ const defaultProps = {
     shouldDisableFocusOptions: false,
 };
 
-class LHNOptionsList extends Component {
-    constructor(props) {
-        super(props);
-
-        this.renderItem = this.renderItem.bind(this);
-        this.getItemLayout = this.getItemLayout.bind(this);
-        this.data = this.props.data;
-    }
+function LHNOptionsList(props) {
+    const data = useMemo(() => props.data, [props.data]);
```

Comment:
I don't understand why `useMemo` is used here. We have a lot of migration issues and I think it's worth to talk about this problem.

Reply:
@s-alves10 could you clarify what the problem with `useMemo` here is?

Reply:
The thought behind this was that if the component was re-rendered, then the entire flat list would re-render because of the change in `data`. By wrapping it in a useMemo hook, I was attempting to limit the number of re-renders. I am relatively new to using the useMemo hook however so please correct me if this is an overuse of it.